### ISSUE TITLE
Custom style spans not being exported

### ISSUE THREAD

**Issue by afamiglietti** [JohnMcLear/ep_custom_styles#4]
Hi John, 

I notice that the span tags generated by the custom styles plugin aren't being exported when I use the etherpad API to call getHTML on a pad where I've inserted some custom styling. 

I'm guessing this means I need to write some code to hook into how etherpad is exporting html, similar to what the folks who wrote the export_authors plugin did? Am I right about that? 

I'm mostly posting this to make sure there isn't some etherpad setting I can use to get it to dump the pad html just as it is saved in the db that I've missed like an idiot. 

Anyway, let me know when you can. Thanks.

**Comment by JohnMcLear** [comment]
Yeah probably, look through the plugin logic and see if its handled or not.  AFAIK I didn't handle export yet but that would be a valuable addition!
